### PR TITLE
Add support for ner tagging

### PR DESCRIPTION
**Pull request by ktagowski**

### CODE REVIEW

File: embeddings/evaluator/sequence_labeling_evaluator.py [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
+    def __init__(
+        self, evaluation_mode: str = "conll", tagging_scheme: Optional[str] = None
+    ) -> None:
+        super().__init__()
+        self.metric = self._get_metric(evaluation_mode, tagging_scheme)
 
+    def _get_metric(
+        self, evaluation_mode: str, tagging_scheme: Optional[str] = None
+    ) -> Union[HuggingFaceMetric, UnitSeqevalMetric]:
+        if evaluation_mode in {"conll", "strict"}:
```
Comment (laugustyniak) [inline embeddings/evaluator/sequence_labeling_evaluator.py:22]: constants for `conll` and `strict`?

File: embeddings/metric/unit_seqeval_metric.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 from embeddings.metric.hugging_face_metric import HuggingFaceMetric
 
 
-class POSTaggingSeqevalMetric(HuggingFaceMetric):
+class UnitSeqevalMetric(HuggingFaceMetric):
```
Comment (laugustyniak) [inline embeddings/metric/unit_seqeval_metric.py:9]: `U-` is from `unit`? :) I didn't know it.

Comment (ktagowski) [reply embeddings/metric/unit_seqeval_metric.py:9]: Yes, in BILOU, U- is from unit

**Review by laugustyniak — APPROVED** [review]

File: tests/test_sequence_labelling.py [hunk 3]
```diff
@@ -37,21 +37,60 @@ def sequence_labeling_pipeline(
...
     ],
 ) -> None:
     flair.set_seed(441)
-    pipeline, path = sequence_labeling_pipeline
+    pipeline, path = ner_tagging_pipeline
     result = pipeline.run()
     path.cleanup()
 
-    np.testing.assert_almost_equal(result["POSTaggingSeqeval"]["overall_f1"], 0.14788732394366197)
+    np.testing.assert_almost_equal(result["seqeval__mode_None__scheme_None"]["overall_f1"], 0)
```
Comment (albert097) [inline tests/test_sequence_labelling.py:96]: i noted to deal with default config and naming

**Pull request merged**